lib/iris/tests/results/cdm/str_repr/cell_methods.__str__.txt

```diff
@@ -8,10 +8,10 @@ air_temperature / (K)               (latitude: 73; longitude: 96)
         pressure                    1000.0 hPa
         time                        1998-12-01 00:00:00
     Cell methods:
-        mean                        longitude (6 minutes, This is a test comment), latitude (12 minutes)
-        average                     longitude (6 minutes, This is another test comment), latitude (15 minutes, This is another comment)
-        average                     longitude, latitude
-        percentile                  longitude (6 minutes, This is another test comment)
+        0                           longitude: latitude: mean: (interval: 6 minutes interval: 12 minutes comment: This is a test comment)
+        1                           longitude: latitude: average: (interval: 6 minutes interval: 15 minutes comment: This is another test comment comment: This is another comment)
```

Comment:
I thought this bit was not correct (though it's hard to read in GitHub), 
but I located the test and saved the cube, and I now see it **_is_** correct !
(it was the old form that was seriously wrong)
What it really means, I'm still not quite sure TBH

But it made me wonder... I know these are really old tests (in tests/test_cdm.py), but I wonder if it's worth adding a cross-check into all these original cases, between what you get when you save the cube + how it prints out, .  
I think you could automate that ?

Reply:
Regretfully shelved due to time constraints.